src/net.cpp

```diff
@@ -1768,7 +1768,13 @@ void CConnman::CreateNodeFromAcceptedSocket(std::unique_ptr<Sock>&& sock,
 {
     int nInbound = 0;
 
-    AddWhitelistPermissionFlags(permission_flags, addr, vWhitelistedRangeIncoming);
+    const bool inbound_onion = std::find(m_onion_binds.begin(), m_onion_binds.end(), addr_bind) != m_onion_binds.end();
```

Comment:
nit, this is just moving code around without modifying it, feel free to ignore; can be written shorter as:

```suggestion
    const bool inbound_onion = std::ranges::find(m_onion_binds, addr_bind) != m_onion_binds.end();
```